<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>951</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_bird_risk_status">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #fffac8; color: #b8860b; font-weight: bold;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>주의</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_a">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RWY A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_b">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RWY B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_b_status">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #ccffcc; color: #228b22; font-weight: bold;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>안전</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_a_status">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #ffcccc; color: #b22222; font-weight: bold;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>위험</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_bird_risk">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>조류 위험도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="map_cctv_stack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>971</width>
     <height>731</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QFrame" name="map_overlay_frame">
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <layout class="QGridLayout" name="overlayStack">
     <item row="3" column="0">
      <widget class="QLabel" name="label_map">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="marker_overlay_placeholder" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_cctv_1">
    <property name="text">
     <string>CCTV 1 영상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cctv_2">
    <property name="text">
     <string>CCTV 2 영상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cctv_3">
    <property name="text">
     <string>CCTV 3 영상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_show_map">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>지도 보기</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_cctv">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CCTV A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CCTV B</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_show_cctv">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>80</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>보기</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="object_area">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>309</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="object_table_page">
    <layout class="QVBoxLayout" name="objectTableLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QTableWidget" name="table_object_list">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>400</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>위치</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종류</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_detail">
       <property name="text">
        <string>상세 보기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="detail_page">
    <layout class="QVBoxLayout" name="detailLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
